package.json

```diff
@@ -37,7 +37,8 @@
   },
   "dependencies": {
     "patch-package": "^6.4.7",
-    "postinstall-postinstall": "^2.1.0"
+    "postinstall-postinstall": "^2.1.0",
+    "tsup": "^4.14.0"
```

Comment:
`tsup` checked all the boxes that `vite` did, + `.d.ts` output.

`Nx` requires a ton of convention, which is **great**, but more than I wanted to change.

`microbundle` is slow because of rollup.